src/fidesctl/cli/commands/core.py

```diff
@@ -48,13 +49,20 @@ def apply(ctx: click.Context, dry: bool, diff: bool, manifests_dir: str) -> None
     "--message",
     help="A message that you can supply to describe the context of this evaluation.",
 )
+@click.option(
+    "-a",
+    "--include-audits",
```

Comment:
```suggestion
    "--audit",
```

Just a nit, I think `include-audit` sounded a little weird and verbose without being descriptive.

Reply:
ya I had a tough time with naming, brevity over descriptive etc. It's sort of an odd fit but happy to go with this for sure 👍🏽